pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -51,9 +51,15 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 		return err
 	}
 
+	conn, err := newGrpcConn(draAddr(endpoint))
+	if err != nil {
+		return err
+	}
+
```

Comment:
Is it possible to create new connection in a lazy way, i.e. only when it's needed?

Reply:
392b4ef8658 introduced the gRPC connection creation only when it is needed and not at the plugin registration time